test/rules/typedef/arrow-call-signature/test.ts.lint

```diff
@@ -18,7 +18,13 @@ var obj = {
 }
 
 setTimeout(() => {}, 1000);
+           ~~       [expected arrow-call-signature to have a typedef]
 setTimeout(function() {}, 1000);
 
+const foo = n => n+1
+          ~~~~       [expected arrow-call-signature to have a typedef]
```

Comment:
why does the failure range extend to the `=`? that seems a little off

Reply:
In `const foo = n => n+1` location.pos = 11
in `const foo = (n => n+1)` location.pos = 13

I didn't touch this part